docs/source/detsim.rst

```diff
@@ -1,5 +1,299 @@
 Detsim
 ==========
 
-.. caution::
-  This page is currently not created, we are working on it. If you would like to contribute on this documentation, reach `me <helena.almamol@gmail.com>`_!
+**Detsim**, from **Detector Simulation**, is a the city that simulates the *detector response* for *fast-simulated* events.
+A *fast-simulated* event consist of a collection of hits of position and energy desposition in the gaseous volume (right figure below). The *detector response* are the signals measured in the light-sensors in the form of time ordered waveforms (left figure below).
+
+We can differenciate two simulation modes implemented in NEXUS: the
```

Comment:
All the information related to fast/full simulations is on the nexus [wiki](https://github.com/next-exp/nexus/wiki/Fast-and-full-simulation ). It can just be mentioned it here, adding a link to the wiki.